jovian/utils/commit.py

```diff
@@ -125,23 +125,30 @@ def commit(message=None,
         log('Failed to detect Jupyter notebook or Python script. Skipping..', error=True)
         return
 
+    # Check for commit from jupyter extension
+    in_jupyter_extension = kwargs.get('jupyter_extension', False)
+
     # Attempt to save Jupyter notebook
-    if in_notebook():
+    if in_notebook() and not in_jupyter_extension:
         save_notebook()
         log('Attempting to save notebook..')
         sleep(1)
 
     # Extract notebook/script filename
     filename = _parse_filename(filename)
     if filename is None:
-        log(FILENAME_MSG)
+        log(FILENAME_MSG, error=True)
+        if in_jupyter_extension:
+            return False
```

Comment:
```suggestion
```

Reply:
there's no difference between `return` and `return False`

Reply:
Makes sense, since we're not using tuples. Thanks